### DISCUSSION TITLE
For original width 1000-pixel page, is it possible to enlarge to 2000 pixel to increase image quality?

### DISCUSSION THREAD

**Discussion by nissansz** [pymupdf/PyMuPDF#2081] [Ideas]
For original width 1000-pixel page, is it possible to enlarge to 2000 pixel to increase image quality?

**Comment by JorjMcKie** [comment]
Don't understand:
"Original" means the page shows an image of that size? If yes then the answer is no.

**Comment by nissansz** [comment]
Ok.
